src/Symfony/Component/Messenger/Transport/TransportFactory.php.orig

```diff
@@ -0,0 +1,83 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
```

Comment:
you don't want to commit this file , nor the .rej one ;)

Reply:
eek, where did those come from :sweat_smile: